server/client.go

```diff
@@ -879,8 +896,10 @@ func (c *client) processMsg(msg []byte) {
 			qsubs := r.qsubs[i]
 			index := c.cache.prand.Intn(len(qsubs))
 			sub := qsubs[index]
-			mh := c.msgHeader(msgh[:si], sub)
-			c.deliverMsg(sub, mh, msg)
+			if sub != nil {
```

Comment:
The array len is non-zero, but I saw a weird panic that I have never seen before where sub was nil